GTG/gtk/editor/editor.py

```diff
@@ -536,14 +614,28 @@ def inserttag_clicked(self, widget):
             self.textview.insert_text("@", itera)
         else:
             self.textview.insert_text(" @", itera)
-        self.textview.grab_focus()
+        # self.textview.grab_focus()
 
     def inserttag(self, widget, tag):
         self.textview.insert_tags([tag])
-        self.textview.grab_focus()
+        # self.textview.grab_focus()
```

Comment:
It's used twice (it'll be used also in def refresh_editor), so I'm leaving it as a separate method for now